networkx/algorithms/community/community_utils.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# utils.py - helper functions for community-finding algorithms
+#
+# Copyright 2011 Ben Edwards <bedwards@cs.unm.edu>.
+# Copyright 2011 Aric Hagberg <hagberg@lanl.gov>.
+# Copyright 2015 NetworkX developers.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+"""Helper functions for community-finding algorithms."""
+
+
+def is_partition(G, communities):
```

Comment:
This function is already in `algorithims/community/quality.py`. It might be a good idea to put it here though...